CHANGELOG.md

```diff
@@ -17,11 +17,14 @@
 - Feedbacks functions to set/get feedback polarity
 - Add in configuration set_velocity_pid, set_position_pid and get_rated_current
 - Add functions to connect to and scan EtherCAT devices using CoE (SDOs).
+- Optional registers for Wizard tests.
+- All registers are initialized for no_connection unit tests.
 
 ### Fixed
 - check_motor_disabled decorator does not work with positional arguments
 - Adapt get_encoded_image_from_dictionary for COM-KIT
 - Feedback test output when symmetry and resolution errors occurred simultaneously.
+- No error because of positioning registers are missing in the dictionary.
```

Comment:
This point can be confusing for external users.

- `No error because of positioning registers are missing in the dictionary.` This error does not happen in previous versions, so, I'm not sure that it should written in the changelog.
- `All registers are initialized for no_connection unit tests` This talks about tests, not about library behavior. This should be added in PR description.
- `Optional registers for Wizard tests.` That's ok, but maybe you mention the backup registers because is not easy to know what you mean by "Optional registers"